xl/metadata/_id3.py

```diff
@@ -143,6 +150,11 @@ def _set_tag(self, raw, tag, data):
             ]
         elif tag == 'WOAR':
             frames = [id3.WOAR(encoding=3, url=d) for d in data]
+        elif tag == 'POPM':
+            #Rating Stars
+            data = int(data[0])
+            rating = self._stars_to_rating(data)
+            frames = [id3.POPM(email='', rating=rating)]
```

Comment:
Other players provide a way to configure the rating email.

Reply:
Here it's the same. Only mp3 supports email in rating.